presto-main/src/main/java/com/facebook/presto/metadata/FunctionRegistry.java

```diff
@@ -356,7 +357,8 @@ public FunctionInfo load(SpecializedFunctionKey key)
                 .functions(MAX_AGGREGATION, MIN_AGGREGATION)
                 .function(COUNT_COLUMN)
                 .functions(ROW_HASH_CODE, ROW_TO_JSON, ROW_EQUAL, ROW_NOT_EQUAL)
-                .function(TRY_CAST);
+                .function(TRY_CAST)
+                .function(CONCAT);
```

Comment:
How about leave `TRY_CAST` last so we don't have to change that line in the diff? :)